modules/services/taffybar.nix

```diff
@@ -29,7 +29,10 @@ in
     systemd.user.services.taffybar = {
         Unit = {
```

Comment:
Maybe also have
```nix
Wants = [ "status-notifier-watcher.service" ]
```
here?

Reply:
What the difference with `After`?

Reply:
The
```nix
Wants = [ "status-notifier-watcher.service" ]
```
says that if you start the taffybar service then the `status-notifier-watcher.service` also should be started. The
```nix
After = [ "status-notifier-watcher.service" ]
```
says that if `status-notifier-watcher.service` and `taffybar.service` is started then the `taffybar.service` will be delayed until `status-notifier-watcher.service` has completed its startup.

So `Wants` and `After` would in this case say, that when taffybar is started then first start status-notifier-watcher 🙂

Reply:
`status-notifier-watcher.service` is not required for taffybar, it is necessary only if you want to use sni tray widget (there is an option to run taffybar with "built-in" status notifier watcher, but it's not recommended, or just without that widget).

May be it would be better to have an option for `taffybar` service like `useStatusNotifierWatcher` to enable `status-notifier-watcher` and put `status-notifier-watcher.service` in `Wants`?

Reply:
As I see it is regular practice to have booth After and Wanted.

Reply:
Sorry for the delay in feedback. Yeah, After and Wanted typically come in a pair.

Now that I'm thinking harder about it and based on what you wrote that `status-notifier-watcher.service` not required for taffybar I believe it might be a good idea to not have
```
Unit.Wants = [ "status-notifier-watcher.service" ];
Unit.After = [ "status-notifier-watcher.service" ];
```
in `taffybar.service` but instead
```
Install.WantedBy = [ "taffybar.service" ];
Unit.Before = [ "taffybar.service" ];
```
in `status-notifier-watcher.service`. I think that more closely matches the indended functionality. Sorry if I'm causing confusion!

Reply:
Done